test/testdata/java_grpc_client_pipeline.baseline

```diff
@@ -1,4 +1 @@
-mkdir -p {OUTPUT}/library-v1-gen-java/src/main/java
-protoc --proto_path=test/fake-repos/gapi-core-proto/src/main/proto/ --proto_path=test/fake-repos/fake-proto --proto_path=MOCK_GRADLE_TASK_OUTPUT --java_out={OUTPUT}/library-v1-gen-java/src/main/java test/fake-repos/fake-proto/fake.proto
```

Comment:
I didn't think about whether the output would be the same at all-- when I test manually, this new version actually produces very different output.